agent/src/softwarecontaineragentadaptor.cpp

```diff
@@ -0,0 +1,58 @@
+#include "softwarecontaineragentadaptor.h"
+
```

Comment:
In the spirit of keeping a consistent code style, move the brackets to the function line.